pkg/controller/nstemplateset/nstemplateset_controller_test.go

```diff
@@ -1277,8 +1276,8 @@ func newIdler(username, name, tier string, options ...objectMetaOption) *toolcha
 		ObjectMeta: metav1.ObjectMeta{
 			Labels: map[string]string{
 				"toolchain.dev.openshift.com/provider":    "codeready-toolchain",
-				"toolchain.dev.openshift.com/tier":        tier,
-				"toolchain.dev.openshift.com/templateref": NewTierTemplateName(tier, "clusterresources", "abcde11"),
+				"toolchain.dev.openshift.com/tier":        "advanced",
+				"toolchain.dev.openshift.com/templateref": NewTierTemplateName("advanced", "clusterresources", "abcde11"),
```

Comment:
I would rather use `basic` as the default hard-coded tier, but it's fine to keep it as it is.

Reply:
well, we always passed `advanced` so I just moved the value here. However, I followed your recommendation and changed to `basic` in cc44404